src/lib.rs

```diff
@@ -403,7 +403,7 @@ pub fn format_string(input: String, config: &Config) -> FileMap {
     file_map
 }
 
-pub fn format(file: &Path, config: &Config) -> FileMap {
+fn format(file: &Path, config: &Config) -> FileMap {
```

Comment:
Could you rename format to format_file please? "format" seems too generic for a function which isn't API